src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -791,8 +791,21 @@ x3dom.registerNodeType(
                         hasColor = false;
                     }
                     this._mesh._numColComponents = numColComponents;
+                    if (fieldName.startsWith("attrib")) {
+                        var attrName = fieldName.slice(7);
```

Comment:
I see. But the name of the attribute is not provided after an underscore ("attrib_temperature") but only in the name field of the attrib child node.